include/nn/oe.h

```diff
@@ -43,5 +43,6 @@ void SetUserInactivityDetectionTimeExtended(bool);
 void FinishStartupLogo();
 nn::settings::LanguageCode GetDesiredLanguage();
 void GetDisplayVersion(DisplayVersion*);
+bool TryPopLaunchParameter(size_t* ,void*, size_t);
```

Comment:
Sorry I messed up the comma in my comment. (If you find a review suggestion weird feel free to ask questions)

```suggestion
bool TryPopLaunchParameter(size_t*, void*, size_t);
```